server/middlewares/index.js

```diff
@@ -18,16 +18,16 @@ exports.isAuth = async (req, res, next) => {
 exports.userById = (req, res, next, id) => {
   User.findById(id)
     .populate('projects')
+    .populate('investments')
     .exec((err, user) => {
       if (err || !user) {
         return res.status(400).json({
           error: 'User not found'
         });
       }
-      const { _id, name, email, about, location, projects, profilePic } = user;
-      req.profile = { _id, name, about, email, projects, location, profilePic };
+      const { _id, name, email, about, location, projects, profilePic, investments } = user;
+      req.profile = { _id, name, about, email, projects, location, profilePic, investments };
```

Comment:
I know this is not new, but this probably could be simplified as `req.profile = user;` Looks kind of odd destructuring and then just mapping all the values again.

Reply:
I do this because returning the user also includes the hashed password

Reply:
I guess and alternative would be to do user.password = null and then return the password as well. This userById method probably needs to be refactored and I don't think it's being used at the moment